memory/main.py

```diff
@@ -238,18 +233,19 @@ def battle_active():
 
 
 def get_current_turn():
-    get_turn_by_index(turn_index = 0)
+    get_turn_by_index(turn_index=0)
```

Comment:
I recall an hour ago seeing this and deciding not to bring it up. Lol. I'm glad you rolled it in.